### DISCUSSION TITLE
AxesLabels for 3D

### DISCUSSION THREAD

**Discussion by gganapat** [WLJSTeam/wljs-notebook#240] [General]
The graphics for 3D plots are awesome, but without the axes labels and tick marks it is difficult to use. I know you mentioned that by the nature of Graphics, it is hard to get these implemented. Are there any plans to fix them? Any workarounds possible? 

gani -

**Comment by JerryI** [comment]
Of course! ❤️
Here is actually sort of [TODO](https://github.com/JerryI/wolfram-js-frontend/blob/default/CONTRIBUTING.md) list, what is planned.
Ticks and axes labels for Graphics3D are definitely possible to implement. Here is [related issue](https://github.com/JerryI/wolfram-js-frontend/issues/216). Feel free to write there anything else related to ticks. 

I think I am going to steal it from Mathics project, because some time ago they managed to draw correctly axes and ticks on 3D canvas. 

I can't promise the exact date, but usually of it takes 2-3 weeks to implement some requested features depending on the difficulty. Anyway, since you have requested it, I will put it on the priority list. :)

As a workaround... You might draw them as lines, but this is cubersome. `Text` primitive for drawing numbers is not supported for now. However, the grid you can turn on from the top-right menu of 3D plot. But without text labels it will probably be useless...

**Reply by gganapat** [reply]
Great! I will await your fixes. I am in no rush. Thank you!
gani -
